src/PeliasMapkitExtensions.swift

```diff
@@ -144,3 +144,10 @@ extension SearchBoundaryRect {
     self.minLatLong = GeoPoint(latitude: maxCoordinate.latitude, longitude: minCoordinate.longitude)
   }
 }
+
+public extension GeoPoint {
```

Comment:
Not sure about the specifics on extensions but I think the issue here is that Pelias as a standalone dependency doesn't know about Tangram classes right?